xla/stream_executor/cuda/cuda_dnn.h

```diff
@@ -713,7 +714,7 @@ absl::StatusOr<CudnnGraph> GetCudnnFlashAttentionOperationGraph(
     std::optional<dnn::TensorDescriptor> page_table_v_descriptor, double scale,
     bool use_dropout, std::optional<double> dropout_rate,
     dnn::FMHAMaskKind mask_type, int sliding_window_length,
-    int max_seg_per_batch);
+    int max_seg_per_batch, std::shared_ptr<ScoreModFunc> score_mod);
```

Comment:
Does this need to be a shared_ptr? To me it looks like a non-owning raw pointer would be enough here.